examples/adafruitio_27_wifimanager/adafruit_27_wifimanager.ino

```diff
@@ -0,0 +1,83 @@
+/* Adafruit IO Example Using WiFiManager
+ *
+ * This is a simple Adafruit feed subscribe example that uses
+ * WiFiManager to handle setup of WiFi credentials and connecting
+ * to the network instead of defining the WiFI SSID and password
+ * explicitly in the code.
+ *
+ * To use this example, add your Adafruit IO Username and Key
+ * and setup a feed called "myfeed".  When you manually add data
+ * to the feed on io.adafruit.com, you'll see that data written to
+ * the serial output.
+ *
+ * Brad Black - 2022
+ *
+ */
+
+#include <WiFiManager.h>
```

Comment:
Add `WiFiManager` as a dependency to:

https://github.com/adafruit/Adafruit_IO_Arduino/blob/master/library.properties#L10

Reply:
WiFiManager would only be a requirement for this example sketch or apps that are going to use it and only applies to ESP8266/ESP32.  So I don't think we want to create a dependancy on WiFiManager for the entire library - it would only be used when needed.  Right?

Reply:
> Brent - do you think we should focus on adding this example with WiFiManager and not make the changes to overload the WiFi functions

If you're able to add an example without modifying function signatures, I'd prefer that.

> I would also be happy to create a guide on this if that is possible.

It's possible! Let's get this PR in first.

> So I don't think we want to create a dependancy on WiFiManager for the entire library - it would only be used when needed. Right?

The dependency in library properties is also actually used by our CI, ci-arduino. The runner instance needs to install the library before testing a sketch. It would still be required.